docs/decisions/0005-user-and-group-management-commands.rst

```diff
@@ -0,0 +1,19 @@
+User and Group Management Commands
+===============================
+
+Context
+-------
+
+Open edX administrators ocassionally need to grant permissions to users for certain django services (e.g. ecommerce, registrar). Right now, this is done manually via django admin of the django app. This method of granting permissions is not endorsed.
+
+The user and group management commands in edx-platform, are supposedly generic enough to be used via a common library/app (such as edx-django-utils) so that users and groups can be managed without much modification to IDAs.
+
+Decision
+--------
+
+User and group management commands should be moved to edx-django-utils from edx-platform and ``edx_django_utils`` should be added to ``INSTALLED_APPS`` section of all django apps so that the user and group management commands are available to the apps.
```

Comment:
```suggestion
User and group management commands should be moved to edx-django-utils from edx-platform and ``edx_django_utils`` should be added to ``INSTALLED_APPS`` section of all django services so that the user and group management commands are available to the services.
```
I recommend saying "services" here, since Django "apps" generally refers to the dozen or so individual apps that make up each IDA.